websites/R/RealLifeGTAV/presence.ts

```diff
@@ -60,15 +60,15 @@ presence.on("UpdateData", async function () {
           set_showUsername,
           " | User: " + document.querySelector("#RPC_Username").textContent
         );
-      presenceData.smallImageKey = "reading";
+
     } else if (document.location.pathname.includes("/faq")) {
       presenceData.details =
         "FAQ" +
         ifSettingEnabled(
           set_showUsername,
           " | User: " + document.querySelector("#RPC_Username").textContent
         );
-      presenceData.smallImageKey = "reading";
```

Comment:
Why remove smallImageKey?

Reply:
I don't like small images

Reply:
You should use either choose to use it all the time in your file, or choose to never use it at all. Imo, you should use it but see if you can find some nice icons.